js/main.js

```diff
@@ -27,6 +27,110 @@ function submitHandler(event) {
   data.entries.unshift(entryValues);
   submitForm.reset();
   imgOnScreen.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $ulEntries.prepend(generateEntriesDOMTree(entryValues));
```

Comment:
Does the view change to the entries view when you submit the form? The app should go to the entries view as soon as the new entry is submitted.